.github/CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# This file contains the code owners for the dbt-core repo.
+# PRs will be automatically assigned for review to the associated 
+# team(s) or person(s) that touches any files that are mapped to them.
+#
+# Consult GitHub documentation for formatting guidelines:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+
+# As a default for areas with no assignment, 
+# the core team as a whole will be assigned
+*       @dbt-labs/core
+
+# Changes to GitHub configurations including Actions
+/.github/ @leahwicz
+
+# Adapter / plugin code
+/plugins @dbt-labs/core-adapters
+
+# Perf regression testing framework
+# This excludes the test project files itself since those aren't specific 
+# framework changes (excluded by not setting an owner next to it- no owner)
+/performance @nathaniel-may
```

Comment:
@nathaniel-may I put you down as the reviewer here for now until we ramp other people up here (which would be the goal). I hope that works for you but if not, let me know